src/gtrends/process.rs

```diff
@@ -58,7 +65,34 @@ pub async fn run(args: &Cli) -> Result<()> {
             .collect::<Result<_>>()?,
     )]);
 
-    println!("{}", serde_json::to_string_pretty(&graphs)?);
+    let victoriametrics: VictoriaMetrics = graphs
+        .get("gtrends")
+        .expect("hard-coded hashmap access of hard-coded entry")
+        .into_iter()
+        .map(|line| VictoriaMetric::try_new("gtrends", "search_term", line))
+        .collect::<Result<_>>()?;
+
+    if let Some(graphs_out) = &args.graphs_out {
+        let mut graphs_out = fs::File::create(graphs_out)?;
+
+        writeln!(
+            &mut graphs_out,
+            "{}",
+            serde_json::to_string_pretty(&graphs)?
+        )?;
+    }
+
+    if let Some(victoriametrics_out) = &args.victoriametrics_out {
+        let mut victoriametrics_out = fs::File::create(victoriametrics_out)?;
+        for victoriametric in victoriametrics {
+            writeln!(
+                &mut victoriametrics_out,
+                "{}",
+                // Output file is jsonlines; this must be a single line (no pretty print)
+                serde_json::to_string(&victoriametric)?
+            )?;
```

Comment:
https://docs.rs/serde_json/latest/serde_json/fn.to_writer.html

replace with 

```
serde_json::to_writer(&mut victoriametrics_out, &vicotriametric)?

// if it doesnt end with a new line (which it should)
writeln!(&mut victoriametrics_out, "")?
```

If these files end up particularly big, you can wrap `victoriametrics_out` in a BufWriter (https://doc.rust-lang.org/std/io/struct.BufWriter.html)

Reply:
Thanks!

Alas, it doesn't end with a newline, so the second line is necessary.